### PR TITLE
Feature/scraper implementation

### PR DESCRIPTION
**Pull request by lynxcs**
Create Scraper Library, which implements IFetcher interface. Will be used later in API

### CODE REVIEW

File: Common/IFetcher.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Common
+{
+    public interface IFetcher
```
Comment (R0PK3) [inline Common/IFetcher.cs:9]: Shouldn't fetcher be async?

Comment (R0PK3) [inline Common/IFetcher.cs:9]: Shouldn't fetcher be async?

Comment (R0PK3) [inline Common/IFetcher.cs:9]: Shouldn't fetcher be async?

**Comment by Afterburn7** [comment]
Maybe don't use Console.WriteLine for logging.

File: Common/IFetcher.cs (hunk 1 above)
Comment (lynxcs) [reply Common/IFetcher.cs:9]: Yup, good point, will fix.

**Comment by lynxcs** [comment]
> Maybe don't use Console.WriteLine for logging.

Yeah agree, but don't having logging yet. Will create ticket for adding logging.

File: Common/IFetcher.cs (hunk 1 above)
Comment (lynxcs) [reply Common/IFetcher.cs:9]: Fixed.

**Comment by Afterburn7** [comment]
Looks fine

**Pull request merged**